### PR TITLE
Alert 6.0.0, 6.0.1, and 6.1.0 support

### PR DESCRIPTION
**Pull request by psantos1113**

### CODE REVIEW

File: src/main/java/com/synopsys/integration/blackduck/installer/dockerswarm/edit/AlertLocalOverridesEditor.java [hunk 1]
```diff
@@ -85,18 +111,26 @@ public void edit(File installDirectory) throws BlackDuckInstallerException {
...
+        if (!alertEncryption.isEmpty() || !customCertificate.isEmpty() || alertDatabase.hasSecrets()) {
+            appendAlertSecrets(ymlBuilder, alertEncryption, customCertificate, alertDatabase);
+            appendSecrets(ymlBuilder, alertEncryption, customCertificate, alertDatabase);
         }
 
-        try (Writer writer = new FileWriter(configFile.getFileToEdit(), true)) {
+        // write the file clean because the original has an alertdb service which may not be there if it's an external deployment.
+        // Also the original file has environment variables set for postgres but if secrets are used those environment variables cannot be present.
+        // To cover all the deployment options the safest thing to do is write the file clean.
+        try (Writer writer = new FileWriter(configFile.getFileToEdit(), false)) {
```
Comment (jamesrichard91) [inline src/main/java/com/synopsys/integration/blackduck/installer/dockerswarm/edit/AlertLocalOverridesEditor.java:128]: @ekerwin  was there a reason that you didn't overwrite the file originally?

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -1,81 +1,68 @@
 base.directory=/srv/blackduck
 install.dry.run=true
-
```
Comment (jamesrichard91) [inline src/main/resources/application.properties:3]: Why did you delete all these spaces?

Comment (psantos1113) [reply src/main/resources/application.properties:3]: I wonder if my save actions in the IDE deleted them.  I will update the file to have the spacing again.

File: src/main/java/com/synopsys/integration/blackduck/installer/dockerswarm/edit/AlertLocalOverridesEditor.java (hunk 1 above)
Comment (ekerwin) [reply src/main/java/com/synopsys/integration/blackduck/installer/dockerswarm/edit/AlertLocalOverridesEditor.java:128]: Yeah, we don't want to replace any of the commented out code that users could edit themselves after the install occurs. We'll need to find a different way to handle alert's overrides to support these versions.
